site-config/hca-dcp/cc-ma-dev/index/projectsEntityConfig.ts

```diff
@@ -33,8 +34,11 @@ export function getMAEntitiesConfig(
   entities: SiteConfig["entities"]
 ): SiteConfig["entities"] {
   return [...entities].map((entity) => {
-    if (entity.route !== "projects") {
-      return entity;
+    if (entity.route === "projects") {
+      return getMAProjectsEntityConfig(entity);
+    }
+    if (entity.route === "files") {
+      return getMAFilesEntityConfig(entity);
     }
     return getMAProjectsEntityConfig(entity);
```

Comment:
I believe the default returned value, will always be `projects` related entity config. This means on the `samples` tab the column config is now `projects` related.

I think the default return should be just `entity`.

Reply:
Thanks! Not sure how I missed that one